test/erc721.js

```diff
@@ -356,8 +326,7 @@ describe('ERC721', () => {
 					const unauthorizedTransferPromise = secondClient.contractCall(compiledContract.bytecode, 'sophia', deployedContract.address, "transferFrom", {
 						args: `(${firstTokenId})`,
 						options: {
-							ttl: config.ttl,
-							gas: config.gas
+							ttl: config.ttl
```

Comment:
'options' repeats to many times .... can you extract it ..if you want to change something you should do more work ... just like now

Reply:
You are right, but for now, I think we don't need to change this because we expect breaking changes in the sdk and the whole "call" structure will probably be different